README.md

```diff
@@ -1,29 +1,56 @@
 # DD2480-CI
-CI is the second assignment in the course DD2480 Software Engineering Fundamentals. The goal of the assignment is to implement a continuous integration server that first clones a repository on GitHub when there are any changes on it, builds the project and notifies the user. 
+CI is the second assignment in the course DD2480 Software Engineering Fundamentals. The goal of the assignment is to implement a continuous integration server that first clones a repository on GitHub when there are any changes pushed on it, builds the project, runs the tests in the project and notifies the user of the results. 
 
 ## Getting Started
-This implementation of a CI-server is made with Java SDK 11 and IntelliJ; therefore make sure you have these installed. The project that uses the CI-server needs to have `Gradle` installed, due to it being used by the CI-server. There must exist a `build.gradle` file in the project. This file must contain the dependencies needed for the project to run.
+This implementation of a CI-server is made with Java SDK 11 and IntelliJ; therefore make sure you have these installed. The project that uses the CI-server needs to have `Gradle` installed, due to it being used by the CI-server. There must exist a `build.gradle` file in the project containing the dependencies needed for the project to build and run.
 
-## Testing
-Tests are written using the `JUnit` library. 
+### Branches
+There are two branches: `master` and `assessment`. The master branch contains the CI-server. The assessment branch is simulating a project with a main class and a simple test for it. The purpose of the assessment branch is to use it to test the functionality of the CI-server. 
+
+## Testing and building locally
+Tests for the CI-server are written using the `JUnit` library. If you want to run the tests locally, there are two ways.
+* **Using the terminal:** first navigate to the the project folder, then use the command `./gradlew test`. To build the project locally use the command `./gradlew build`. For windows machines you can use `gradlew test/build`.
```

Comment:
The build command also run the tests. Maybe we can differantiate both like : To only run the test -> `gradle test` but to `gradle build` build and run the tests